src/assets/scss/templates/_doppler-plus-plans-form.scss

```diff
@@ -62,7 +62,7 @@
 
     .field-item.dp-validate,
     .field-item.error {
-      margin: variables.$dp-spaces--lv0;
+      // margin: variables.$dp-spaces--lv0;
```

Comment:
What do you think about removing this commented line or adding a clarification message?

Reply:
Yes, I delete it!